src/pages/SearchPage/Heatmap/Info/Info.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  InfoContainer,
+} from './Info.style';
+import InfoRow from '../InfoRow/InfoRow';
+
+function Info({ info }) {
+  const [selected, setSelected] = useState({});
+
+  return (
+    <InfoContainer>
+      <InfoRow selected={selected} setSelected={setSelected} day={0} hours={info[0]} />
```

Comment:
You can use an array here. Actually `info` is already an array right? Can't you just do `info.map(...` here?